### ISSUE TITLE
Direct deposit image: incorrect alt text; accessibilityRole not announced

### ISSUE THREAD

**Issue by bessgreen** [department-of-veterans-affairs/va.gov-team#25832]
## What happened?
When a screen reader is enabled, the check image on the Direct Deposit edit form makes the following announcement:

"You can find your 9-digit routing number on the bottom left side of a check. You can find your account number in the bottom center of a check. **Your address. Date. Routing check number number number. Account.**"

Issues:
1) ~The bold section of the above phrase is incorrect and should not be read at all~ 6/17 update: this is a result of the [Text Recognition Toggle](https://www.iphoneincanada.ca/news/ios-14-voiceover-recognition-photos/)
2) the accessibilityRole ("image") is not announced at all

## Specs:
iPhone 11

## Steps to Reproduce
1. Enable screen reader
2. Navigate to Direct Deposit edit form
3. Select image and observe announcement

## Desired behavior
1. Accessibility requirements for the check image:
     - accessibilityRole= image.
     - Alt text: "You can find your 9-digit routing number on the bottom left side of a check. You can find your account number in the bottom center of a check"
2. Both the accessibilityRole and alt text should be announced when the device's screen reader is on

## Acceptance Criteria
See desired behavior

## Bug Severity
- 4 - Low

## Linked to Story
#13907 - see Accessibility Implementation Guidelines

## Screen shot(s) and additional information
Navigate to the secure messaging area.
This is mocked out with user +1414.

**Comment by digitalken** [comment]
PASS 6/25/2021 - alt text and image announced.  Closing.
